airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -159,6 +161,18 @@ def convert_schema_into_query(
 
         return query_template
 
+    @staticmethod
+    def serialize_protobuf_message(message) -> str:
+        """
+        This method tryies to serialize a given object into JSON if the given object is a protocol bugger message.
+        If not, the normal str() is applied. That helps keep the complex objects written as parsable structures.
+        """
+
+        try:
+            return json_format.MessageToJson(message._pb).replace('\n', '')
```

Comment:
Because we are passing not an instance of a Repeated or RepeatedComposite, but whatever is repeated items inside those are, I cannot read the public Pb property of those.

Unless we would consider serialization of the entire object without drilling into its elements.